File: Python/List-2/sum13.py
def sum13(nums):
  sum = 0
  found13 = False
  
  for i in range(len(nums)):
    if (nums[i] == 13):
      found13 = True
    
    if not found13:
      sum += nums[i]
    elif (i - 1 >= 0 and nums[i - 1] == 13):
      found13 = False
      
  return sum

File: Java/Array-2/sum13.java
public int sum13(int[] nums) {
  boolean seen13 = false;
  int sum = 0;
  for (int i = 0; i < nums.length; i++) {
    if (nums[i] != 13) {
      if (seen13) seen13 = false;
      else sum += nums[i];
    } else {
      seen13 = true;
    }
  }
  return sum;
}
